tests/resources/splash/readme.md

```diff
@@ -0,0 +1,12 @@
+This directory contains splash responses (and the respective HAR) that can be obtained with the splash docker container.
```

Comment:
Do we still need splash responses?

Reply:
Yes, we use them in some tests to mock playwright: the `mock_playwright` context loads the splash json and fills out the Content class such that communication with playwright is skipped. Theoretically we could just never mock playwright, because it seems to be running nice in the github pipeline, but then we have test dependencies on external websites which I would like to avoid because they are out of our control.

Reply:
Mmmh, I like mocking playwright, due to hexagonal architecture, e.g. 
To me it does not matter where this HAR comes from. If I know it is a set of mocked data in the shape of a HAR file, that is enough.

Reply:
Yep - I agree!  Then I'll leave it as is. Currently, the `Content` class (mainly) hides away the playwright internals. And we mock playwright in most of the unittests with this checked in data which is nice:-)
The `test_content.py` file is the only part where we actually test that the integration with playwright works which is also nice to have as part of the unittest pipeline :-)

Reply:
I suggest to rename "splash responses" to "responses as HAR files".

Reply:
done!